discord_bots/utils.py

```diff
@@ -62,6 +65,460 @@
 DELTA_UPPER_UNICODE = "\u03B4"
 
 
+def buildCategoryOutput(category: Category) -> str:
```

Comment:
use underscore_case like the other functions. Maybe something  like `build_category_str` or `get_category_str`. Whatever you want

Reply:
I believe this was added in a PR after this was originally raised, so was likely missed from my review.
Good catch, have renamed it here, and in all references.